version/1/4/fr/code_of_conduct.md

```diff
@@ -46,32 +46,31 @@ comportements qu'ils jugent inappropriés, menaçants, injurieux, ou nuisibles.
 
 ## Objectifs
 
-Ce Code de Conduite s'applique à la fois au sein des espaces du projet
-ainsi que dans les espaces publics lorsqu'un individu représente le projet
-ou sa communauté. Font parties des exemples de représentation d'un projet ou 
-d'une communauté le fait d'utiliser une adresse email propre au projet, de
-poster sur les réseaux sociaux avec un compte officiel, ou d'intervenir comme
-représentant programmé à un événement en-ligne ou hors-ligne. La représentation
-du projet pourra être autrement définie et clarifiée par les mainteneurs du
-projet.
+Ce Code de Conduite s'applique à la fois au sein des espaces de projet
+ainsi que dans les espaces publics quand un individu représente le projet
+ou sa communauté. Les exemples de représentation d'un projet ou d'une
+communauté incluent le fait d'utiliser une adresse email propre au projet,
+de poster sur les réseaux sociaux avec un compte officiel, ou d'intervenir
+comme représentant programmé à un événement en-ligne ou hors-ligne. La
+représentation du projet pourra être autrement définie et clarifiée par
+les mainteneurs du projet.
 
 ## Application
 
 Les cas de comportements abusifs, harcelants ou tout autre comportement
 inacceptable peuvent être signalés en contactant l'équipe du projet à
 [INSERER ADRESSE EMAIL]. Toutes les plaintes seront examinées et étudiées
-et se traduiront par une réponse appropriée aux
+et se traduiront par une réponse appropriée et appropriée aux
 circonstances. L'équipe du projet s'engage à garder confidentielles les
 informations de la personne qui remonte un incident. Plus de détails sur
 la politique de mise en application des règles peuvent être publiés séparément.
 
 Les mainteneurs du projet qui ne suivent ou qui n'appliquent pas le Code de
-Conduite de bonne foi s'exposent temporairement ou de façon permanente à des
+Conduite de bonne foi, s'exposent temporairement ou de façon permanente à des
```

Comment:
The comma here is a good idea, thanks :)

Reply:
``